### PR TITLE
restructuring THREEx location-based-only

### PR DESCRIPTION
**Pull request by kalwalt**
**⚠️ All PRs have to be done versus 'dev' branch, so be aware of that, or we'll close your issue ⚠️**

**What kind of change does this PR introduce?**

No real changes in the code, it improve the location-based-threex and prepare to load it as a npm module.

**Can it be referenced to an Issue? If so what is the issue # ?**
see discussion on issue #357
**How can we test it?**

Test the `three.js/examples/location-based`
**Summary**

three.js location-based bundle file wasn't in build file so it make difficult to publish it on npm. This PR solve that issue.
**Does this PR introduce a breaking change?**
No at all.
**Please TEST your PR before proposing it. Specify here what device you have used for tests, version of OS and version of Browser**
Tested on Ubuntu 20.04 64bit, Chrome browser 97.0.4692.71.
Mobile device:
Oppo A72 running Android 11, Chrome browser 97.0.4692.87 

**Other information**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -85,12 +85,6 @@ module.exports = (env, argv) => {
       libraryTarget: 'umd',
       globalObject: 'this'
     },
-    resolve: {
```
Comment (kalwalt) [inline webpack.config.js:88]: @nickw1 i think this is not needed for aframe-location-only dist lib, tell me if i'm right.

Comment (nickw1) [reply webpack.config.js:88]: @kalwalt will look at this in the next couple of days, sorry have had a busy week!

Comment (kalwalt) [reply webpack.config.js:88]: @nickw1 no problem! The code should be ok if you can test or just take a look at, it would be nice. :slightly_smiling_face:

Comment (nickw1) [reply webpack.config.js:88]: @kalwalt done! Sorry for the delay, was difficult to find the time last week.

Comment (kalwalt) [reply webpack.config.js:88]: ok so i will publish a new npm package!

**Pull request merged**